### PR TITLE
Cpp polish 8

### PR DESCRIPTION
**Pull request by jtlap**

### CODE REVIEW

File: test/unit/module/real/core/is_not_equal.cpp [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  using eve::detail::map;
+  using eve::as;
+  using v_t = eve::element_type_t<T>;
+  using ui_t = eve::as_integer_t<T, unsigned>;
+
+  TTS_EQUAL(eve::definitely(eve::is_not_equal)(T(1), T(1)), eve::false_(eve::as<T>()) );
+  TTS_EQUAL(eve::definitely(eve::is_not_equal)(T(1), v_t(1) ), eve::false_(eve::as<T>()) );
+  TTS_EQUAL(eve::definitely(eve::is_not_equal)(T(3), T(1)), eve::true_(eve::as<T>()));
+  TTS_EQUAL(eve::definitely(eve::is_not_equal)(T(3), v_t(1) ), eve::true_(eve::as<T>()));
+  using eve::as;
```
Comment (jfalcou) [inline test/unit/module/real/core/is_not_equal.cpp:92]: duplicate line

File: test/unit/module/real/core/is_not_flint.cpp [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+};
+
+//==================================================================================================
+// Tests for eve::is_not_flint
+//==================================================================================================
+auto mfo2 = []<typename T>(eve::as_<T> const & tgt)
+{
+  return eve::maxflint(tgt);
+};
+auto mf2 = []<typename T>(eve::as_<T> const & tgt)
```
Comment (jfalcou) [inline test/unit/module/real/core/is_not_flint.cpp:41]: blank line needed here, not at line 43 ;)

File: test/unit/module/real/core/is_not_greater.cpp [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+  using eve::logical;
+  using eve::as;
+  using v_t = eve::element_type_t<T>;
+  using ui_t = eve::as_integer_t<T, unsigned>;
+
+  TTS_EQUAL(eve::almost(eve::is_not_greater)(T(1) , T(1)   ), eve::true_(eve::as<T>()) );
+  TTS_EQUAL(eve::almost(eve::is_not_greater)(T(1) , v_t(1) ), eve::true_(eve::as<T>()) );
+  TTS_EQUAL(eve::almost(eve::is_not_greater)(T(3) , T(1)   ), eve::false_(eve::as<T>()));
+  TTS_EQUAL(eve::almost(eve::is_not_greater)(T(3) , v_t(1) ), eve::false_(eve::as<T>()));
+  using eve::as;
```
Comment (jfalcou) [inline test/unit/module/real/core/is_not_greater.cpp:88]: duplicate

File: test/unit/module/real/core/is_not_greater_equal.cpp [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+//==================================================================================================
+#include "test.hpp"
+#include <eve/constant/valmin.hpp>
+#include <eve/constant/valmax.hpp>
+#include <eve/function/is_not_greater_equal.hpp>
+#include <eve/function/fuzzy/is_not_greater_equal.hpp>
+#include <eve/constant/eps.hpp>
+#include <eve/constant/mindenormal.hpp>
+#include <eve/logical.hpp>
+#include <cmath>
```
Comment (jfalcou) [inline test/unit/module/real/core/is_not_greater_equal.cpp:16]: why cmath ?

File: test/unit/module/real/core/is_not_greater_equal.cpp [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+  using eve::detail::map;
+  using eve::as;
+  using v_t = eve::element_type_t<T>;
+  using ui_t = eve::as_integer_t<T, unsigned>;
+
+    TTS_EQUAL(eve::definitely(eve::is_not_greater_equal)(T(1), T(1)   ), eve::false_(eve::as<T>()) );
+    TTS_EQUAL(eve::definitely(eve::is_not_greater_equal)(T(1), v_t(1) ), eve::false_(eve::as<T>()) );
+    TTS_EQUAL(eve::definitely(eve::is_not_greater_equal)(T(1), T(3)   ), eve::true_(eve::as<T>()));
+    TTS_EQUAL(eve::definitely(eve::is_not_greater_equal)(T(1), v_t(3) ), eve::true_(eve::as<T>()));
+    using eve::as;
```
Comment (jfalcou) [inline test/unit/module/real/core/is_not_greater_equal.cpp:79]: duplicate

File: test/unit/module/real/core/is_not_less.cpp [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+**/
+//==================================================================================================
+#include "test.hpp"
+#include <eve/constant/valmin.hpp>
+#include <eve/constant/valmax.hpp>
+#include <eve/function/is_not_less.hpp>
+#include <eve/function/fuzzy/is_not_less.hpp>
+#include <eve/constant/eps.hpp>
+#include <eve/logical.hpp>
+#include <cmath>
```
Comment (jfalcou) [inline test/unit/module/real/core/is_not_less.cpp:15]: why cmath ?

File: test/unit/module/real/core/is_not_less.cpp [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+  using eve::logical;
+  using eve::as;
+  using v_t = eve::element_type_t<T>;
+  using ui_t = eve::as_integer_t<T, unsigned>;
+
+    TTS_EQUAL(eve::almost(eve::is_not_less)(T(1), T(1)  ), eve::true_(eve::as<T>()) );
+    TTS_EQUAL(eve::almost(eve::is_not_less)(T(1), v_t(1)), eve::true_(eve::as<T>()) );
+    TTS_EQUAL(eve::almost(eve::is_not_less)(T(1), T(3)  ), eve::false_(eve::as<T>()));
+    TTS_EQUAL(eve::almost(eve::is_not_less)(T(1), v_t(3)), eve::false_(eve::as<T>()));
+    using eve::as;
```
Comment (jfalcou) [inline test/unit/module/real/core/is_not_less.cpp:88]: duplicate

File: test/unit/module/real/core/is_not_less_equal.cpp [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+**/
+//==================================================================================================
+#include "test.hpp"
+#include <eve/constant/valmin.hpp>
+#include <eve/constant/valmax.hpp>
+#include <eve/function/is_not_less_equal.hpp>
+#include <eve/function/fuzzy/is_not_less_equal.hpp>
+#include <eve/constant/eps.hpp>
+#include <eve/logical.hpp>
+#include <cmath>
```
Comment (jfalcou) [inline test/unit/module/real/core/is_not_less_equal.cpp:15]: why cmath ? 
copy paste much ?

File: test/unit/module/real/core/is_not_less_equal.cpp [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+  using eve::detail::map;
+  using eve::as;
+  using v_t = eve::element_type_t<T>;
+  using ui_t = eve::as_integer_t<T, unsigned>;
+
+  TTS_EQUAL(eve::definitely(eve::is_not_less_equal)(T(1)                     , T(1)     )   , eve::false_(eve::as<T>()) );
+  TTS_EQUAL(eve::definitely(eve::is_not_less_equal)(T(1)                     , v_t(1)   )   , eve::false_(eve::as<T>()) );
+  TTS_EQUAL(eve::definitely(eve::is_not_less_equal)(T(3)                     , T(1)     )   , eve::true_(eve::as<T>()));
+  TTS_EQUAL(eve::definitely(eve::is_not_less_equal)(T(3)                     , v_t(1)   )   , eve::true_(eve::as<T>()));
+  using eve::as;
```
Comment (jfalcou) [inline test/unit/module/real/core/is_not_less_equal.cpp:78]: duplicate

File: test/unit/module/real/core/is_not_real.cpp [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+**/
+//==================================================================================================
+#include "test.hpp"
+#include <eve/constant/valmin.hpp>
+#include <eve/constant/valmax.hpp>
+#include <eve/constant/false.hpp>
+#include <eve/function/is_not_real.hpp>
+#include <eve/constant/smallestposval.hpp>
+#include <eve/logical.hpp>
+#include <cmath>
```
Comment (jfalcou) [inline test/unit/module/real/core/is_not_real.cpp:15]: why cmath

File: test/unit/module/real/core/is_pow2.cpp [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+  SPDX-License-Identifier: MIT
+**/
+//==================================================================================================
+#include "test.hpp"
+#include <eve/concept/value.hpp>
+#include <eve/constant/valmin.hpp>
+#include <eve/constant/valmax.hpp>
+#include <eve/function/is_pow2.hpp>
+#include <eve/function/firstbitset.hpp>
+#include <bit>
```
Comment (jfalcou) [inline test/unit/module/real/core/is_pow2.cpp:14]: why bit ?

File: test/unit/module/real/core/is_unordered.cpp [hunk 12]
```diff
@@ -0,0 +1,66 @@
...
+/**
+  EVE - Expressive Vector Engine
+  Copyright : EVE Contributors & Maintainers
+  SPDX-License-Identifier: MIT
+**/
+//==================================================================================================
+#include "test.hpp"
+#include <eve/constant/nan.hpp>
+#include <eve/function/is_unordered.hpp>
+#include <cmath>
```
Comment (jfalcou) [inline test/unit/module/real/core/is_unordered.cpp:11]: why cmath ?

**Review by jfalcou — CHANGES_REQUESTED** [review]

File: test/unit/module/real/core/is_not_greater_equal.cpp (hunk 4 above)
Comment (jtlap) [reply test/unit/module/real/core/is_not_greater_equal.cpp:16]: all changes done

**Comment by jtlap** [comment]
j'ai répondu à tes demandes

**Review by jfalcou — APPROVED** [review]

**Pull request merged**